File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/main/fragment/view/MainView.java
package ua.meugen.android.pdfcomposer.ui.activities.main.fragment.view;

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.view.MvpView;

public interface MainView extends MvpView {

    void createNewDocument();

    void viewRecentDocuments();
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/main/fragment/view/MainView.kt
package ua.meugen.android.pdfcomposer.ui.activities.main.fragment.view

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.view.MvpView

interface MainView : MvpView {

    fun createNewDocument()

    fun viewRecentDocuments()
}
